src/utils/redirectUtils.ts

```diff
@@ -10,10 +16,34 @@ export const redirectToLoginPage = () => {
   }
 };
 
-export const redirectToErrorPage = () => {
+const redirectToErrorPage = () => {
   if (import.meta.env.PROD) {
     window.location.assign("/error");
   } else {
     console.warn("Prevented redirect to /error page");
   }
 };
+
+const handleRedirectIfNotValid = (userResult: UserResult, envs?: Envs) => {
+  if (userResult.statusCode === HttpStatusCode.Unauthorized) {
+    // this means jwt is expired
+    // remove it to perform redirect to login page
+    const cookies = new Cookies();
+    cookies.remove("jwt");
```

Comment:
Is this a decision that you can make? It is a tldraw cookie then yes, otherwise you can only redirct and let handle it by the owner of this cookie.

Reply:
fixed